spec/README.md

```diff
@@ -24,6 +24,16 @@ The source of this diagram can be found [here](../images/envelope_relationships.
 The [validation model] provides pseudocode showing how these layers fit
 together. See the [documentation] for more background and examples.
 
+## Tagged Releases
+
+The latest [tagged release] version matches the [SemVer](https://semver.org)
+MAJ.MIN version of the Attestation Framework spec.
+
+Backwards-compatible semantic updates to the spec (except predicates) are
+indicated through new tagged minor version releases.
+We use new tagged patch version releases to indicate updates to predicate
```

Comment:
```suggestion

Updates to predicate specifications (backwards-compatible or not), and _backwards-compatible_ changes to the language bindings are also indicated via minor version releases.
```

Reply:
Apparently I don't know how to make the suggestion span multiple lines because it should take line 35 as well. :/

Reply:
Ah so what you're suggesting is different from what we had been doing so far. So far. we've been bumping the patch version to indicate changes in everything but the statement layer spec to keep the major/minor version in line with the core spec. What you're suggesting seems to be that we might release a `v1.2` release even though we haven't touched the statement layer? I don't have a strong preference, though my sense is that the tagged release shouldn't diverge from the core spec version, mostly as a signal to users.

Reply:
Oh!  Then I just misunderstood, I didn't mean to change your suggestion.  Perhaps to avoid confusion user UPPERCASE for 'PATCH' and provide some examples?

Reply:
Can do! Examples would definitely be a good addition.

Reply:
Done.